### PR TITLE
Updates names

### PR DESCRIPTION
**Pull request by stepps00**
Will replace https://github.com/whosonfirst-data/whosonfirst-data-admin-ro/pull/18/files

This PR takes the commits from #18 and updates some name properties to align with how Who's On First handles variant names, and exportifies records to validate them before merging.

@mihaicosareanu-bolt - take a look and let me know if you have any questions.

### CODE REVIEW

File: data/101/793/711/101793711.geojson [hunk 1]
```diff
@@ -106,7 +106,7 @@
     "wof:lang_x_spoken":[
         "ron"
     ],
-    "wof:lastmodified":1593203628,
+    "wof:lastmodified":1603739906,
```
Comment (stepps00) [inline data/101/793/711/101793711.geojson:109]: No changes for this record, will be removing this change before merge.

File: data/101/835/591/101835591.geojson [hunk 2]
```diff
@@ -95,7 +95,7 @@
     "wof:lang_x_spoken":[
         "ron"
     ],
-    "wof:lastmodified":1593203628,
+    "wof:lastmodified":1603739907,
```
Comment (stepps00) [inline data/101/835/591/101835591.geojson:98]: No changes for this record, will be removing this change before merge.

File: data/101/835/595/101835595.geojson [hunk 3]
```diff
@@ -121,7 +121,7 @@
     "wof:lang_x_spoken":[
         "ron"
     ],
-    "wof:lastmodified":1593203627,
+    "wof:lastmodified":1603739907,
```
Comment (stepps00) [inline data/101/835/595/101835595.geojson:124]: No changes for this record, will be removing this change before merge.

File: data/101/835/597/101835597.geojson [hunk 4]
```diff
@@ -101,7 +101,7 @@
     "wof:lang_x_spoken":[
         "ron"
     ],
-    "wof:lastmodified":1593203628,
+    "wof:lastmodified":1603739908,
```
Comment (stepps00) [inline data/101/835/597/101835597.geojson:104]: No changes for this record, will be removing this change before merge.

File: data/112/603/972/9/1126039729.geojson [hunk 5]
```diff
@@ -96,7 +96,7 @@
         }
     ],
     "wof:id":1126039729,
-    "wof:lastmodified":1588368337,
+    "wof:lastmodified":1603739909,
```
Comment (stepps00) [inline data/112/603/972/9/1126039729.geojson:99]: No changes for this record, will be removing this change before merge.

**Review by mihaicosareanu-bolt — APPROVED** [review]
The changes look good, I guess they will solve my issue, however, I will still state that a lot of the variants there are actually invalid from a Romania admin perspective. If this prevents a future automatic import to overwrite them or something, then it's great 👍

**Comment by mihaicosareanu-bolt** [comment]
Another tiny reminder here as it will greatly help me to have the new names 🙏

**Comment by stepps00** [comment]
Thanks for your help and patience with this, @mihaicosareanu-bolt!

**Pull request merged**